src/foam/core/reflow/cmd/Commands.js

```diff
@@ -781,3 +780,149 @@ foam.CLASS({
     }
   ]
 });
+
+
+foam.CLASS({
+  package: 'foam.core.reflow.cmd',
+  name: 'Button',
+  extends: 'foam.core.reflow.cmd.Command',
+
+  classes: [
+    {
+      name: 'FlowAction',
+      extends: 'foam.lang.Action',
+      documentation: 'Small inner class to set up some basic view and configuration settings to make actions easier to use in console, might want to move this out if we ever want to use them outside these commands',
```

Comment:
small ish thing - readability is important. Please try to maintain 80 char line length.

Here use ` and do multi line